src/main/java/Entities/Output.java

```diff
@@ -0,0 +1,114 @@
+package Entities;
+
+import java.util.HashMap;
+
+/**
+ * Output Entity stores all the output data that is to be presented to the user each move
+ */
+public class Output {
+    /**
+     * InstanceVar stateOutput: A hashmap that contains predefinied outputs for specific states
+     * InstanceVar finalOutput: the string object containing the cumulative output
+     */
+    private HashMap<String, String> stateOutput;
+    private String finalOutput;
+
+    /**
+     * The constructor for this class, it pre-defines all the key-value pairs in the hashmap
+     * note: some values are null as those values do not present any output to the user and are simply processing stages
+     * note: some keys start off as empty string as those change based on logic, so empty strings are used as placeholders
+     * note: the Value with respect to each key represents the output giving the option for the user to choose what they want to do
+     * i.e. the output string value is to determine the next state from the current state
+     */
+    public Output(){
+        this.stateOutput = new HashMap<>();
+        this.stateOutput.put("InitialMenu", "Welcome to Monopoly++, Would you like to ");
```

Comment:
Would it be better to store these strings in a txt file and have a helper which reads each line and adds it to the class? I think you may need to create an interface and whatnot but I'm not sure if this is the best way to populate the HashMap.

Reply:
Made a new class and moved the string into a file